packages/abi-gen-wrappers/CHANGELOG.json

```diff
@@ -9,6 +9,10 @@
             {
                 "note": "Update wrappers to include `getABIEncodedTransactionData` for view and pure functions",
                 "pr": 1863
+            },
+            {
+                "note": "Initial support for Python",
+                "pr": 1878
```

Comment:
`abi-gen-wrappers` is the package for the Typescript wrappers. Why is there a CHANGELOG entry in here related to Python?

Reply:
Discussed elsewhere.  I'm going to create a new folder under `packages` for this functionality to live in.